java/mapreduce/src/test/org/apache/orc/mapred/TestOrcFileEvolution.java

```diff
@@ -235,35 +235,37 @@ public void testListEvolution() {
   @Test
   public void testPreHive4243CheckEqual() {
     // Expect success on equal schemas
-    checkEvolution("struct<_col0:int,_col1:string>",
+    checkEvolutionPosn("struct<_col0:int,_col1:string>",
                    "struct<_col0:int,_col1:string>",
                    struct(1, "foo"),
-                   struct(1, "foo", null), false, addSarg, false);
+                   struct(1, "foo", null),
+                   false, addSarg, false);
   }
 
   @Test
   public void testPreHive4243Check() {
     // Expect exception on strict compatibility check
     thrown.expectMessage("HIVE-4243");
-    checkEvolution("struct<_col0:int,_col1:string>",
+    checkEvolutionPosn("struct<_col0:int,_col1:string>",
                    "struct<_col0:int,_col1:string,_col2:double>",
                    struct(1, "foo"),
                    struct(1, "foo", null), false, addSarg, false);
   }
 
   @Test
   public void testPreHive4243AddColumn() {
-    checkEvolution("struct<_col0:int,_col1:string>",
+    checkEvolutionPosn("struct<_col0:int,_col1:string>",
                    "struct<_col0:int,_col1:string,_col2:double>",
                    struct(1, "foo"),
-                   struct(1, "foo", null), true, addSarg, false);
+                   struct(1, "foo", null),
+                   true, addSarg, false);
```

Comment:
Let's keep the original form to reduce the patch size.

Reply:
Done